ui/components/multichain/app-header/app-header.js

```diff
@@ -306,12 +314,20 @@ export const AppHeader = ({ onClick }) => {
               padding={2}
               gap={2}
             >
-              <PickerNetwork
-                label={currentNetwork?.nickname}
-                src={currentNetwork?.rpcPrefs?.imageUrl}
-                onClick={() => dispatch(toggleNetworkMenu())}
-                className="multichain-app-header__contents__network-picker"
-              />
+              {popupStatus ? null : (
+                <Box
+                  display={DISPLAY.FLEX}
+                  className="multichain-app-header__contents__network-container"
+                >
```

Comment:
Grid should handle this
```suggestion
                <div>
```